src/pages/Organization/Orglogin.tsx

```diff
@@ -1,44 +1,83 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState, useRef } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 function Orglogin() {
+  const [url, setURL] = useState('');
+
+  const [errMsg, setErrMsg] = useState('');
+  const [success, setSuccess] = useState(false);
+
+  const errRef: any = useRef();
+
+  const navigate = useNavigate();
+
+  const URL_REGEX =
+    /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=\+\$,\w]+@)?[A-Za-z0-9.-]+|(?:www.|[-;:&=\+\$,\w]+@)[A-Za-z0-9.-]+)((?:\/[\+~%\/.\w-_]*)?\??(?:[-\+=&;%@.\w_]*)#?(?:[\w]*))?)/;
```

Comment:
Currently, when someone enters andela.pulse.com as a url, they get an error that is invalid. Try to change the regex to match urls with the pulse domain.

Reply:
Actually we should automatically append the `.pulse.com` on the input value to improve user experience